src/__tests__/calculator.test.tsx

```diff
@@ -0,0 +1,95 @@
+import calculate from '../';
+
+describe('calculate works with valid inputs', () => {
+  it('returns the correct value when passed the simplest expression', () => {
+    expect(calculate('1')).toBe(1);
+  });
+
+  it('returns the correct value when passed simple equations with two numbers', () => {
+    expect(calculate('2 + 2')).toBe(4);
+  });
+
+  it('returns the correct value when passed equations with negative numbers', () => {
+    expect(calculate('(2 + -2)')).toBe(0);
+  });
```

Comment:
Notice how a lot of these `it` statements start with the same set of words. Whenever you see this happening, it's a good place to create another `describe` block to reduce redundancy and also create logically similar tests

In this case, I would expect:

1. `describe('returns the correct value when...', () => {`
2. `describe('does not throw an error when...', () => {`